bgpd/bgp_nexthop.c

```diff
@@ -734,17 +735,67 @@ bool bgp_subgrp_multiaccess_check_v4(struct in_addr nexthop,
 	return false;
 }
 
+static void bgp_show_bgp_path_info_flags(uint32_t flags, json_object *json)
+{
+	json_object *json_flags = NULL;
+	if (!json)
+		return;
+
+	json_flags = json_object_new_array();
+	if (CHECK_FLAG(flags, BGP_PATH_IGP_CHANGED))
+		json_array_string_add(json_flags, "igpChanged");
+	if (CHECK_FLAG(flags, BGP_PATH_DAMPED))
+		json_array_string_add(json_flags, "damped");
+	if (CHECK_FLAG(flags, BGP_PATH_HISTORY))
+		json_array_string_add(json_flags, "history");
+	if (CHECK_FLAG(flags, BGP_PATH_SELECTED))
+		json_array_string_add(json_flags, "selected");
+	if (CHECK_FLAG(flags, BGP_PATH_VALID))
+		json_array_string_add(json_flags, "valid");
+	if (CHECK_FLAG(flags, BGP_PATH_ATTR_CHANGED))
+		json_array_string_add(json_flags, "attrChanged");
+	if (CHECK_FLAG(flags, BGP_PATH_DMED_CHECK))
+		json_array_string_add(json_flags, "dmedCheck");
+	if (CHECK_FLAG(flags, BGP_PATH_DMED_SELECTED))
+		json_array_string_add(json_flags, "dmedSelected");
+	if (CHECK_FLAG(flags, BGP_PATH_STALE))
+		json_array_string_add(json_flags, "stale");
+	if (CHECK_FLAG(flags, BGP_PATH_REMOVED))
+		json_array_string_add(json_flags, "removed");
+	if (CHECK_FLAG(flags, BGP_PATH_COUNTED))
+		json_array_string_add(json_flags, "counted");
+	if (CHECK_FLAG(flags, BGP_PATH_MULTIPATH))
+		json_array_string_add(json_flags, "multipath");
+	if (CHECK_FLAG(flags, BGP_PATH_MULTIPATH_CHG))
+		json_array_string_add(json_flags, "multipathChg");
```

Comment:
nit: I suggest `multipathChanged` be as `attrChanged`, `igpChanged`. Because now it will look very inconsistent and weird.